tests/installation/encrypted_volume_activation.pm

```diff
@@ -0,0 +1,27 @@
+use base "basetest";
+use strict;
+use testapi;
+
+sub run {
+    assert_screen 'encrypted_volume_activation_prompt';
+    if (get_var('ENCRYPT_CANCEL_EXISTING')) {
+	send_key 'alt-c', 5;
```

Comment:
where did you get this from? see https://github.com/os-autoinst/os-autoinst/blob/master/testapi.pm#L1068. In short, the `5` does not make sense